src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -349,7 +355,7 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
         <AssetHeader>{asset.name ?? `${asset.collectionName} #${asset.tokenId}`}</AssetHeader>
       </AssetInfoContainer>
       {isOwner ? (
```

Comment:
if walletAsset can be undefined should check here (like: isOwner && walletAsset ? )

Reply:
walletAsset being undefined is part of the isOwner check